Mathlib/Algebra/Category/ModuleCat/Monoidal/Basic.lean

```diff
@@ -171,11 +172,60 @@ instance monoidalCategory : MonoidalCategory (ModuleCat.{u} R) := MonoidalCatego
 instance : CommRing ((𝟙_ (ModuleCat.{u} R) : ModuleCat.{u} R) : Type u) :=
   inferInstanceAs <| CommRing R
 
+theorem carrier_tensorObj {M N : ModuleCat.{u} R} :
+    (M ⊗ N).carrier = TensorProduct R M.carrier N.carrier :=
+  rfl
+
+@[simp]
+theorem hom_tensorHom {K L M N : ModuleCat.{u} R} (f : K ⟶ L) (g : M ⟶ N) :
+    (f ⊗ g).hom = TensorProduct.map f.hom g.hom :=
+  rfl
+
+@[simp]
+theorem hom_whiskerLeft (L : ModuleCat.{u} R) {M N : ModuleCat.{u} R} (f : M ⟶ N) :
+    (L ◁ f).hom = f.hom.lTensor L :=
+  rfl
+
+@[simp]
+theorem hom_whiskerRight {L M : ModuleCat.{u} R} (f : L ⟶ M) (N : ModuleCat.{u} R) :
+    (f ▷ N).hom = f.hom.rTensor N :=
+  rfl
+
+@[simp]
+theorem hom_hom_leftUnitor {M : ModuleCat.{u} R} :
+    (λ_ M).hom.hom = (TensorProduct.lid _ _).toLinearMap :=
+  rfl
+
+@[simp]
+theorem hom_inv_leftUnitor {M : ModuleCat.{u} R} :
+    (λ_ M).inv.hom = (TensorProduct.lid _ _).symm.toLinearMap :=
+  rfl
+
+@[simp]
+theorem hom_hom_rightUnitor {M : ModuleCat.{u} R} :
+    (ρ_ M).hom.hom = (TensorProduct.rid _ _).toLinearMap :=
+  rfl
+
+@[simp]
+theorem hom_inv_rightUnitor {M : ModuleCat.{u} R} :
+    (ρ_ M).inv.hom = (TensorProduct.rid _ _).symm.toLinearMap :=
+  rfl
+
+@[simp]
+theorem hom_associator_hom {M N K : ModuleCat.{u} R} :
```

Comment:
I can't ask you to reassociate the tokens in the name, but I can ask you to commute them!
```suggestion
theorem hom_hom_associator {M N K : ModuleCat.{u} R} :
```